elide-core/src/main/java/com/yahoo/elide/core/sort/Sorting.java

```diff
@@ -50,13 +50,9 @@ public Sorting(final Map<String, SortOrder> sortingRules) {
     public <T> boolean hasValidSortingRules(final Class<T> entityClass,
                                         final EntityDictionary dictionary) throws InvalidValueException {
 
-        final List<String> entities = dictionary.getAttributes(entityClass);
-        sortRules.keySet().stream().forEachOrdered(sortRule -> {
-            if (!entities.contains(sortRule) && !JSONAPI_ID_KEYWORD.equals(sortRule)) {
-                throw new InvalidValueException(entityClass.getSimpleName()
-                        + " doesn't contain the field " + sortRule);
-            }
-        });
+        //This validates the paths for the given entityClass or throws InvalidValueException
+        getValidSortingRules(entityClass, dictionary);
```

Comment:
At a glance, it's not obvious why this method performs a `getValidSortingRules` and doesn't use the result. I do see that this functionality is embedded in the logic of `getValidSortingRules`, but you have to dig to find it. Can we restructure the code a bit so it's clearer?